scripts/jenkins-yoctobuild-build.sh

```diff
@@ -125,7 +157,7 @@ for build in deployments deviceadm deviceauth inventory useradm; do (
 
 if [ "$CLEAN_BUILD_CACHE" = "true" ]
 then
-    sudo rm -rf /mnt/sstate-cache/*
+    sudo rm -rf /mnt/sstate-cache/
```

Comment:
Why do you need this? I think this might have some bad effects, because you delete the mount point.